container/src/main/java/org/jboss/hal/testsuite/container/Browser.java

```diff
@@ -43,9 +46,12 @@ public static Browser instance() {
     private RemoteWebDriver webDriver;
 
     private Browser() {
-        super();
+        super(DockerImageName.parse("quay.io/redhatqe/selenium-standalone")
+                .asCompatibleSubstituteFor("selenium/standalone-chrome"));
         withNetwork(Network.INSTANCE)
-                .withCapabilities(new ChromeOptions())
+                .withCapabilities(new ChromeOptions()
+                        .addArguments("--no-sandbox", "--disable-dev-shm-usage")
+                        .setHeadless(true))
```

Comment:
Does this work for both [environments](https://github.com/hal/manatoko#test-environment): local and remote?

Reply:
Only works for the `remote` environment. When executing in `local`,  the web driver + selenium is taken care of by arquillian graphene

Reply:
Cool. LGTM!